.github/workflows/test.yml

```diff
@@ -145,7 +145,7 @@ jobs:
       - name: Maven Test Build
         run: >
           mvn -T2 -B --no-snapshot-updates
-          -D skipChecks
+          -D skipChecks -D surefire.rerunFailingTestsCount=3
```

Comment:
we also need to enable the profile `extract-flaky-tests` then?

Reply:
Correct - for some reason the client tests were already doing that :see_no_evil: